From e733c24baf0c87e0b3c45351ff916c5f3a652826 Mon Sep 17 00:00:00 2001
From: Benitoite <kd6kxr@gmail.com>
Date: Sun, 16 Dec 2018 22:18:37 -0800
Subject: [PATCH] Define EXIV2_TEST_VERSION by including exiv2.hpp

AppleClang 10 did not allow compilation without exiv2/exiv2.hpp or exiv2/version.hpp
Also, version.hpp says use exiv2.hpp

* 2018-12-29 asturm@gentoo.org: Backported to 2.5.1
---
 src/TransplantExif/TransplantExifDialog.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/TransplantExif/TransplantExifDialog.cpp b/src/TransplantExif/TransplantExifDialog.cpp
index c38a51e4..1fac2e0f 100644
--- a/src/TransplantExif/TransplantExifDialog.cpp
+++ b/src/TransplantExif/TransplantExifDialog.cpp
@@ -26,6 +26,7 @@
 #include <QFileInfo>
 #include <image.hpp>
 #include <exif.hpp>
+#include <exiv2/exiv2.hpp>
 
 #include "Common/global.h"
 #include "Common/config.h"
